src/store/actions/food.js

```diff
@@ -0,0 +1,33 @@
+export const ADD_FOOD = 'ADD_FOOD';
+export const DELETE_FOOD = 'DELETE_FOOD';
+export const INCREMENTE = 'INCREMENTE';
+export const DECREMENTE = 'DECREMENTE';
+
+export const addFood = (payload) => (dispatch) => {
+  console.log('addddddd');
```

Comment:
Always remember to remove console.log after debugging

Reply:
Deleted line code console.log('adddddd');